liquigraph-spring-boot-starter/src/test/java/org/liquigraph/spring/LiquigraphAutoconfigurationTest.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright 2014-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.liquigraph.spring;
+
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+import javax.sql.DataSource;
+import org.h2.jdbcx.JdbcDataSource;
+import org.junit.After;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.liquigraph.spring.starter.LiquigraphAutoConfiguration;
+import org.liquigraph.spring.starter.LiquigraphDataSource;
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.graphdb.Result;
+import org.neo4j.harness.junit.Neo4jRule;
+import org.slf4j.bridge.SLF4JBridgeHandler;
+import org.springframework.boot.test.util.EnvironmentTestUtils;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class LiquigraphAutoconfigurationTest {
+
+    static {
+        SLF4JBridgeHandler.removeHandlersForRootLogger();
+    }
+
+    @ClassRule
+    public static final Neo4jRule neo4j = new Neo4jRule();
+
+    @After
+    public void prepare() {
+        try (Result ignored = graphDb().execute("MATCH (n) DETACH DELETE n")) { }
+    }
+
+    @Test
+    public void runs_migrations_with_bean_configured_by_properties() {
+        try (AnnotationConfigApplicationContext context = new AnnotationConfigApplicationContext()) {
+            loadContext(context, "liquigraph.url=jdbc:neo4j:" + neo4j.httpURI().toString());
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "liquigraph.url=jdbc:neo4j:" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)